spring-boot/admin/src/main/resources/application.yml

```diff
@@ -12,7 +12,8 @@ logging:
 ygg:
   mail:
     # Define if CoatRack should send verification emails to newly registered users
-    verify-new-users-via-mail: true
+    # TODO Feature is not working, yet. After #53 is fixed, the parameter can be set to "true".
```

Comment:
I am pretty sure that feature works fine, I used it in the past. However, I agree it probably makes sense to switch it off in the default setting, because for local dev testing of CoatRack the verification feature is not required in most cases.